WalletWasabi.Fluent/Models/Wallets/HardwareWalletModel.cs

```diff
@@ -35,6 +45,7 @@ public async Task<bool> AuthorizeTransactionAsync(TransactionAuthorizationInfo t
 		}
 		catch (Exception ex)
 		{
+			// TODO: In Coldcard case, the error could be to higher fee rate, so we should ask the user to lower it.
```

Comment:
Can you remove this and create and issue instead?

Reply:
OFC
there is the issue:
https://github.com/zkSNACKs/WalletWasabi/issues/12346